Фронт переписан с использованием TailwindCSS вместо Bootstrap

BetterLuckNextLife opened the pull request.

BetterLuckNextLife committed:
Добавлена поддержка tailwindcss

BetterLuckNextLife committed:
Убраны файлы bootstrap, перенесены templates

PanterPard committed:
Обновлена страница [login]

Разметка переписана под Tailwind CSS.

BetterLuckNextLife committed:
Обновлён requriments.txt

BetterLuckNextLife committed:
Полностью переписан фронт

BetterLuckNextLife committed:
Убраны загрузки bootstrap

PanterPard commented:
Нашёл баг с входом: после регистрации пользователя нельзя войти, хотя все поля заполнены верно. Также присутствует баг с focus-границей у полей ввода: отображаются две границы разных цветов.

BetterLuckNextLife committed:
Работоспособность входа восстановлена

PanterPard committed:
Улучшен дизайн страниц

Улучшен вид карточек, форм и таблицы. Удалены лишние контейнеры, упрощены некоторые классы. Изменена цветовая схема (экспериментально). В [.gitignore] временно добавлены некоторые файлы.

PanterPard committed:
Добавлены проверки на поля ввода
